.github/workflows/tests.yml

```diff
@@ -126,6 +121,11 @@ jobs:
       - name: pnpm install
         run: pnpm install --frozen-lockfile
 
+      - name: Copy official plugins
+        if: matrix.target == 'plugins'
+        run: |
+          cp -r /var/www/discourse/plugins/* plugins/
```

Comment:
There is a tiny problem here but I think it is OK. If we remove an official plugin, it will not be removed from our build until a new `discourse/discourse_test:release` image is built.

Reply:
That will definitely come back to bite us. Perhaps we should update the rake task to include a 'prune' option which removes any unrecognised plugins?

Reply:
That is an option as well 👍 Will work towards that.